cmd/registry-experimental/cmd/compute/descriptor.go

```diff
@@ -47,7 +53,7 @@ func descriptorCommand(ctx context.Context) *cobra.Command {
 				log.FromContext(ctx).WithError(err).Fatal("Failed to get client")
 			}
 			// Initialize task queue.
-			taskQueue, wait := core.WorkerPool(ctx, 64)
+			taskQueue, wait := core.WorkerPool(ctx, 1)
```

Comment:
Should we still use the worker pool if we're only processing one task at a time? Although the abstraction makes it easy to use, creating a goroutine and passing tasks around on a channel seems like a lot of complexity if our goal is sequential execution.

Reply:
I expect to raise this in the future with a --jobs flag